redirectionio/proxy-symfony/0.1/config/packages/redirectionio.yaml

```diff
@@ -0,0 +1,7 @@
+# redirection.io Configuration
+redirection_io:
+    connections:
+        # add here all your connection options as follow:
```

Comment:
I would remove this comment. The config is already saying that.
And I would also remove the next comments **or** I would add the following comment:
`#another_agent: unix:///var/run/redirectionio.sock`